docs/about/how_can_i_try_mautic.rst

```diff
@@ -0,0 +1,8 @@
+How can I try Mautic?
+#####################
+
+Mautic is available to download for free [:xref:`Download Mautic`], allowing you to explore its full capabilities without restrictions when you host it yourself.
```

Comment:
```suggestion
Mautic is available to download for free, just head over to the website where you can :xref:`Download Mautic`, allowing you to explore its full capabilities without restrictions when you host it yourself.
```

If you check the link file for this link, it outputs the words 'download Mautic' so in this context it doesn't really make sense how it's written.